src/filtrering/filtrering-mine-filter/filtrering-mine-filter.tsx

```diff
@@ -1,26 +1,36 @@
 import React from 'react';
-import {useSelector} from 'react-redux';
-import {AppState} from '../../reducer';
-import LagredeFilterInnhold from "./mine-filter_innhold";
-import {AlertStripeFeil} from "nav-frontend-alertstriper";
-import {HandlingsType} from "../../ducks/mine-filter";
-import {STATUS} from "../../ducks/utils";
+import { useSelector } from 'react-redux';
+import { AppState } from '../../reducer';
+import LagredeFilterInnhold from './mine-filter_innhold';
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import { HandlingsType, MineFilter } from '../../ducks/mine-filter';
+import { STATUS } from '../../ducks/utils';
 
 function FiltreringMineFilter(props: { filtergruppe: string }) {
     const mineFilterState = useSelector((state: AppState) => state.mineFilter);
     const mineFilter = mineFilterState.data;
-    const sortertMineFilter = mineFilter.sort((a, b) => a.filterNavn.toLowerCase()
-        .localeCompare(b.filterNavn.toLowerCase(), undefined, {numeric: true}));
+    mineFilter.sort((a: MineFilter, b: MineFilter) => {
```

Comment:
there is same code in sidevelger.tsx, so maybe you can extract this sorting logic to some util function and call that function here.

Reply:
This is code for the old designe, maybe better to just remove it?

Reply:
its still used, until we release the new design to everyone.

Reply:
Drag and drop is disabled on the old designe. It should not be hard to implement it for the old design though

Reply:
yep, we can check with Kari is it necessary to implement it in old design...I suppose its not.

Reply:
Ok i sent a message 👍